mis_builder_report_xlsx/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017-2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "MIS Builder - Report XLSX",
+    "summary": "Glue module for MIS Builder and Report XLSX"
+    " that allow to export MIS report in XSLX format.",
+    "version": "18.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV, GRAP, Odoo Community Association (OCA)",
```

Comment:
Do feel it's important to add GRAP as author here. In a way it is not "new" as it is shuffling code around.